canonical-web-and-design/jp.ubuntu.com
# Update to focal

## Done

- Update talisker v0.18.0
- Update docker image to focal
- Update to use gevent

## QA

- `DOCKER_BUILDKIT=1 docker build --build-arg BUILD_ID=test --tag jp.ubuntu.com  .`
-  `docker run -ti -p "8006:80" --env SECRET_KEY=secret_key jp.ubuntu.com`
- http://0.0.0.0:8006/
- Play around with the site and make sure it works well
